assets/src/components/p4_button/edit.js

```diff
@@ -15,107 +16,71 @@ import classnames from 'classnames';
  */
 import { __ } from '@wordpress/i18n';
 import { useCallback, useState } from '@wordpress/element';
-import { compose } from '@wordpress/compose';
 import {
+	Button,
```

Comment:
You've added these but they seem unused...?

Reply:
yes indeed. removing unused stuff 👍